geocity/apps/permits/admin.py

```diff
@@ -1318,6 +1332,20 @@ def sortable_str(self, obj):
     sortable_str.short_description = _("3.2 Enquêtes public")
 
 
+class DuoConfigAdmin(admin.ModelAdmin):
```

Comment:
Add integrator mixing. Integrator should be able to configure DUO for his organization only

Reply:
Done